new_experiment/nbconverted/debug.py

```diff
@@ -0,0 +1,405 @@
+
+
+# In[40]:
+
+
+template_nonrecount2_nofilter_DE_stats_filename = os.path.join(
+    DE_stats_dir,
+    "DE_stats_template_data_cis-par-KU1919_real_without_filtering.txt"
+)
+
+template_nonrecount2_nofilter_DE_stats = pd.read_csv(
+    template_nonrecount2_nofilter_DE_stats_filename,
+    sep="\t",
+    index_col=0,
+    header=0
+)
+
+template_nonrecount2_nofilter_DE_stats.loc["RBPJL"]
+
+
+# In[41]:
+
+
+template_nonrecount2_nofilter_DE_stats.loc["SSTR5.AS1"]
+
+
+# **Takeaways:**
+# * Case 1: genes with only simulated statistics is because template experiment has all 0 counts
+# * Case 2: genes with fewer than 25 simulated experiments, some simulated experiments have all 0 counts for those genes
+# * Case 3 (only found using non-recount2 template): genes with missing p-value in template experiment have NaN output from DESeq2, which indicates those genes that have been automatically filtered by DESeq2 as very likely to not have no significance: https://bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html#pvaluesNA
```

Comment:
Interesting! So it turned out that none of these cases were due to issues with name resolution/name mapping, just due to how DESeq is handling things or cases with all 0 gene counts?

Reply:
Yes. I assumed that there would be some issue with the name mapping given that this issue only appeared in the analysis using recount2 data. And the pipelines where similar expect this mapping step so that was my initial guess. But yes it does look like the issues are with DESeq. Which I think is an easier problem to solve compared to having to track down any mapping issues that can be a bit of a rabbit hole.